twitchobserver/twitchobserver.py

```diff
@@ -122,6 +122,12 @@ def subscribe(self, callback):
 
         self._subscribers.append(callback)
 
+    def unsubscribe(self, callback):
+        """Unsubscribe a callback from the observer."""
+
+        if callback in self._subscribers:
+            del self._subscribers[self._subscribers.index(callback)]
```

Comment:
I think it is a bit simpler(and I do prefer more simple 😄 ). Otherwise I do believe it is functionally the same.